### DISCUSSION TITLE
Tip Jar on User Profile

### DISCUSSION THREAD

**Discussion by binoyxj** [deso-protocol/dips#87] [Medium CIPs]
We've diamonds and Creator Coins to support a creator on BitClout. But there are times when you'd like to support somone in a more impactful way. 

By adding a tipping feature on the user profile and allowing anyone to drop some $CLOUT may change the dynamics. 

The send $CLOUT feature exists, and this proposal is all about making it a more prominent feature encouraging easy tipping. This could even be turned into a recurring monthly tip. 

PS: As suggested by @MeredithMarsone

**Comment by tijno** [comment]
This i think is fairly easy to implement but would be even better in combo with the "msg" change being discussed here #46 #81

Also theres discussion in the diamonds clout proposal which is related here #26

**Comment by ItsAditya-xyz** [comment]
I can imagine tipJar as a link to sending creator coint or CLOUT
